charts/fluentd-elasticsearch/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.serviceMonitor.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
```

Comment:
Some older version of prometheus operator requires the serviceMonitor to be create in the namespace of the operator so it would be a good idea to add it here using a conditional.